src/components/documents/renderers/DocumentMetaRenderer.tsx

```diff
@@ -0,0 +1,23 @@
+import { McfFamilyMeta } from "@components/document/McfFamilyMeta";
+import { DocumentMeta } from "../DocumentMeta";
+import { getApprovedYearFromEvents } from "@helpers/getApprovedYearFromEvents";
+
+const MultilateralClimateFunds = "MCF";
```

Comment:
Make constants descriptive and in all uppercase so that they are easier to spot in the code please